tests/test_file_service.py

```
@@ -0,0 +1,392 @@
+
+        result = await save_upload_file_async(mock_upload_file, "test.txt")
+
+        mock_get_running_loop.assert_called_once()
+        mock_loop.run_in_executor.assert_called_once_with(ANY, mock_save_upload_file, mock_upload_file, "test.txt", None)
+        assert result == "/mock/path/file.txt"
+
+    @pytest.mark.asyncio
+    @patch('services.file_service.save_upload_file_with_meta')
+    @patch('asyncio.get_running_loop')
+    @patch('services.file_service._get_thread_pool') # Added patch
+    async def test_save_upload_file_with_meta_async(self, mock_get_running_loop, mock_save_upload_file_with_meta, mock_upload_file, event_loop):
```

Ruff (0.14.0)

Unused method argument: `event_loop`

(ARG002)